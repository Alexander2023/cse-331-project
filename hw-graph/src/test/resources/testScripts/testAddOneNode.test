# Create a graph
CreateGraph graph1

# Add a single node
AddNode graph1 n1

# Prints the node in the graph
ListNodes graph1