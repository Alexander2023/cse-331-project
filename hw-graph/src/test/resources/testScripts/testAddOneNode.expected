# Create a graph
created graph graph1

# Add a single node
added node n1 to graph1

# Prints the node in the graph
graph1 contains: n1